### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by vmisai**
[DEMO LINK](https://vmisai.github.io/layout_moyo-header/)
[TEST REPORT LINK](https://vmisai.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1,3 +1,66 @@
...
+  list-style: none;
+}
+
+li {
+  margin-left: 20px;
+}
+
+li:first-child {
+  margin-left: 0;
+}
```
Comment (etojeDenys) [inline src/style.css:35]: try to avoid tag selectors, better to add class names in html

File: src/style.css [hunk 2]
```diff
@@ -1,3 +1,66 @@
...
+
+a:hover {
+  color: var(--blue-color);
+}
+
+a.is-active {
+  color: var(--blue-color);
+}
+
```
Comment (etojeDenys) [inline src/style.css:56]: we can join it using the comma

File: src/index.html [hunk 3]
```diff
@@ -7,8 +7,26 @@
...
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <img class="logo" src="images/logo.png"
+      alt="Logo">
```
Comment (etojeDenys) [inline src/index.html:17]:
```suggestion
      <img 
        class="logo" 
        src="images/logo.png"
        alt="Logo"
      >
```

**Review by etojeDenys — CHANGES_REQUESTED** [review]

**Review by maxim2310 — APPROVED** [review]
Good work!!!

**Pull request closed without merging**